client/src/GlobalStyles.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Global, css } from "@emotion/core";
+import ArchivoBlack from "./assets/fonts/ArchivoBlack-Regular.ttf";
+import Roboto from "./assets/fonts/Roboto-Regular.ttf";
+
+const GlobalStyles = () => {
+  return (
+    <Global
+      styles={css`
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+          margin: 0;
+        }
+        :root {
+          --font-color-orange: #ffa200;
+          --font-color-white: #fffdfd;
+          --page-background-color: #262524;
+          --contrast-color-anthracite: #34322c;
+          --font-archivoblack: "ArchivoBlack-Regular";
+          --font-roboto: "Roboto-Regular";
+        }
+
+        html {
+          @font-face {
+            font-family: "ArchivoBlack";
+            src: url(${ArchivoBlack}) format("truetype");
+          }
+          @font-face {
+            font-family: "Roboto";
+            src: url(${Roboto}) format("truetype");
+          }
+          font-family: "Roboto";
```

Comment:
I think it is common to list the `font-family` in the `body` section.

Reply:
will do so! thx